app/views/hyrax/homepage/_home_marketing_boxes.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="row home-marketing-boxes">
+  <div class="col-md-3 col-xs-6 marketing-box">
+    <%= image_tag("icons/magazine.png", class: 'img-responsive', alt: 'magazine icon') %>
+    <div class="marketing-caption">
+      Fulfill funder and journal requirements
```

Comment:
Should this kind of content be housed in the translations file?

Reply:
@mikedaul Great point! I'll do that and resubmit...